src/sugar3/graphics/toggletoolbutton.py

```diff
@@ -112,19 +147,39 @@ def set_palette_invoker(self, palette_invoker):
         type=object, setter=set_palette_invoker, getter=get_palette_invoker)
 
     def set_tooltip(self, text):
+        '''
+        Sets the tooltip of the toogle tool button. Displays when
+        user hovers over the button with cursor.
+        Args:
+            tooltip (string): tooltip to be added to the button
+        '''
         self.set_palette(Palette(text))
 
     def set_accelerator(self, accelerator):
+        '''
+        Sets keyboard shortcut that activates this button
+        Args:
+            accelerator(string): accelerator to be set. Should be in
+            form <modifier>Letter
+        Example:
+        def set_accelerator(self, 'accel')
```

Comment:
Same as above. Your example of invoking should be: class_name.method(arguments). Here ,you have defined a function, not invoked it.